src/deluge/gui/l10n/l10n.cpp

```diff
@@ -5,6 +5,31 @@
 namespace deluge::l10n {
 Language const* chosenLanguage = nullptr;
 
+std::string_view getView(const deluge::l10n::Language& language, deluge::l10n::String string) {
```

Comment:
why did you move these?

Reply:
@PaulFreund provided me with an l10n patch to add to my PR. It was to reduce the binary size impact of the string localization